src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentAdd.tag

```diff
@@ -7,8 +7,19 @@
 <%@ attribute name="firstIndex" %>
 <%@ attribute name="secondIndex" %>
 <%@ attribute name="thirdIndex" %>
-<c:set var="divId" value="${firstIndex}-${secondIndex}-${thirdIndex}" />
-<c:set var="divIdAsJsParams" value="${firstIndex},${secondIndex},${thirdIndex}" />
+<%@ attribute name="fourthIndex" %>
+
+<c:choose>
+    <c:when test="${not empty firstIndex && not empty secondIndex && not empty thirdIndex && not empty fourthIndex}">
+        <c:set var="divId" value="${fourthIndex}-${firstIndex}-${secondIndex}-${thirdIndex}" />
+        <c:set var="divIdAsJsParams" value="${firstIndex},${secondIndex},${thirdIndex}, { sectionIndex: ${fourthIndex} }" />
+    </c:when>
+    <c:when test="${not empty firstIndex && not empty secondIndex && not empty thirdIndex}">
```

Comment:
This seems a bit more complicated than it should be. I might be missing something, but are we able to simplify this entire `c:choose` to a check if `fourthIndex` is not empty?